src/motor-operations.cc

```diff
@@ -85,11 +86,40 @@ static char test_acceleration_ok(float acceleration) {
 }
 #endif
 
+typedef struct HistoryPositionInfo {
+  HistoryPositionInfo () : position_steps(0), sign(1) {}
+  int position_steps;
```

Comment:
Can this be more described ?
I think the 'position_steps' is the absolute machine position at the end of the move (important to describe as this is different from the delta steps the queue gets).
So the fraction is also good to describe (1/0xffffffff-th of the difference between the previous step and now)